root/components/area-map.tt

```diff
@@ -0,0 +1,40 @@
+<div id="largemap"></div>
+<script type="text/javascript">//<![CDATA[
+
+    [%~ DEFAULT places=entities ~%]
+    var map,
+        places = [];
+
+    [%~ FOR place IN places ~%]
+        [%~ IF place.coordinates.defined ~%]
+            places.push({
+                title: '[% place.name | js %]',
+                type: '[% place.l_type_name %]',
+                lat: [% place.coordinates.latitude %],
+                lon: [% place.coordinates.longitude %]
+            });
+        [%~ END ~%]
+    [%~ END ~%]
```

Comment:
I guess there is a cleaner way to transform a "places" data structure server-side into a javascript array, but I didn't find it